tests/test_artifactory_commands.py

```diff
@@ -25,25 +25,36 @@ def conan_test():
     os.chdir(current)
     run("conan profile detect")
     run("conan remove '*' -c")
-    run(f'conan remote add extensions-stg {os.getenv("ART_URL")}/api/conan/extensions-stg')
-    run(f'conan remote add extensions-prod {os.getenv("ART_URL")}/api/conan/extensions-prod')
+
+    out = run("conan remote list")
+
+    if "extensions-stg" not in out:
+        run(f'conan remote add extensions-stg {os.getenv("ART_URL")}/api/conan/extensions-stg')
+    
+    if "extensions-prod" not in out:
+        run(f'conan remote add extensions-prod {os.getenv("ART_URL")}/api/conan/extensions-prod')
+    
+    run(f'conan remote login extensions-stg "{os.getenv("CONAN_LOGIN_USERNAME_EXTENSIONS_STG")}" -p "{os.getenv("CONAN_PASSWORD_EXTENSIONS_STG")}"')
+    run(f'conan remote login extensions-prod "{os.getenv("CONAN_LOGIN_USERNAME_EXTENSIONS_PROD")}" -p "{os.getenv("CONAN_PASSWORD_EXTENSIONS_PROD")}"')
```

Comment:
Had to change this because apparently there's some problem when login via environment variables, will open an issue in Conan repo to check that